brigade-worker/test/k8s.ts

```diff
@@ -186,8 +188,8 @@ describe("k8s", function() {
           // We also have to check this for the sidecar pod. But right now this is
           // embedded in a string.
           let sidecar = jr.runner.spec.initContainers[0]
-          assert.equal(sidecar.env.length, 5)
-          assert.equal(sidecar.env[4].name,"BRIGADE_REPO_KEY", "Has BRIGADE REPO KEY as param")
+          assert.equal(sidecar.env.length, 6)
+          assert.equal(sidecar.env[5].name,"BRIGADE_REPO_KEY", "Has BRIGADE REPO KEY as param")
```

Comment:
^^ These tests were broken on master. When a new env var was added in a recent PR, this test did not get updated.